devel/eric6/Portfile

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            eric6
+version         19.02
+revision        1
```

Comment:
The revision starts at 0, not 1.

Reply:
@mf2k 
Hi, I amended the Portfile as per your comments. I'm new to GitHub workflow, how can I submit the new file? Should I open a new PR or can I use this one?
Thanks